packages/kbn-check-mappings-update-cli/current_mappings.json

```diff
@@ -4022,4 +4022,4 @@
     "dynamic": false,
     "properties": {}
   }
-}
+}
```

Comment:
Removing the new line here was possibly not intentional 🙂

Reply:
>Removing the new line here was possibly not intentional 🙂 

It is CI bot running script, can't do much here :)